notebook/services/contents/filemanager.py

```diff
@@ -284,7 +286,9 @@ def _dir_model(self, path, content=True):
                         self.log.warning("Error stat-ing %s: %s", os_path, e)
                     continue
 
-                if not stat.S_ISREG(st.st_mode) and not stat.S_ISDIR(st.st_mode):
+                if (not WINDOWS_CONTAINER
```

Comment:
I certainly appreciate aiming for minimal impact. I'm coming from the perspective that anything WINDOWS_CONTAINER specific is going to get minimal testing, so if we can reduce the number of cases it will be easier for us to maintain / catch regressions. Plus, I think showing broken links should be fine, as long as it doesn't cause an error in directory listing (using lstat fixes this, I believe).

Instead of gating on Windows, since it can be a link, maybe it should be adding `stat.S_ISLNK` to the checks instead of making it conditional on the platform. Does that return True for these mounts? What is `os.stat(path).st_mode` for these mounts?